bin/reportdaily.py

```diff
@@ -56,6 +64,142 @@ class MissingSubCommand(ValueError):
+                          'team': team, 'current_day': today_date, 'start_year': year}
+
+    # create a config file
+    os.makedirs(os.path.dirname(CONIFGPATH), exist_ok=True)
+    with open(CONIFGPATH, 'w') as user_config:
+        config.write(user_config)
+    print(f"The file was created at this path {CONIFGPATH}")
+    show_config()
+
+
+def show_config():
+    """Show the configs to the user"""
+
+    # read the config
+    parser = ConfigParser()
+    parser.read(f"{CONIFGPATH}")
+
+    print("Your current configuration at the moment")
+    print(f"""
+    Name: {parser.get('settings','name')}
+    Team: {parser.get("settings", "team")}
+    Date: {parser.get("settings", "current_day")}
+    Year: {parser.get("settings", "start_year")}
+
+    If you desire to make changes to the configuration try the -c or --change option for the init command
+    """)
+    # Team: {parser.get("settings", "team")}
+    # Team: {parser.get("settings", "team")},
+    # Date: {parser.get("settings", "current_day")},
+    # Year: {parser.get("settings", "start_year")}")
+
+
+def check_if_config_exists(args):
+    """Check if the config files exists and if not it creates a new config file"""
+
+    try:
+        with open(f'{CONIFGPATH}') as user_config:
+            print("Configs already exists")
+            show_config()
+            if args.change == True:
+                change_config(args)
+    except FileNotFoundError:
+        print("Config File does not exist")
+        create_config(args)
+
+
+def change_config(args):
+    """Allows the user to change direct his existing config file when the -c/--change option is true"""
+
+    # all user options
+    choice_table = {"Name": "t1", "Team": "t2", "Year": "t3"}
+    tmp_input = ''
+    overwrite_input = ' '
+    change_data = ''
+
+    # show and ask user what he wants to overwrite
+    print("""
+        "What do you want to change?"
+        Your options are
+        Name
+        Team
+        Year
+        """)
+
+    # check for right user input --> keeps going until user enters the right dict
+    keep_going = True
+    while(keep_going):
+        tmp_input = input("Name, Team, Year? ")
+        if tmp_input in choice_table:
+            print(f"{tmp_input} exists in key")
+            # need to map the keys  right to the settings --> from Name to name
+            if tmp_input == "Name":
+                tmp_input = "name"
+            elif tmp_input == "Team":
+                tmp_input = "team"
+            elif tmp_input == "Year":
+                tmp_input = "start_year"
+
+            overwrite_input = input("Enter the change ")
+            keep_going = False
+        else:
+            print("No key in config found --> Try again")
+            keep_going = True
+
+    # add config parser to file
+    config = ConfigParser()
+    config.read(f"{CONIFGPATH}")
+    config.set("settings", f"{tmp_input}", f"{overwrite_input}")
+    with open(f"{CONIFGPATH}", "w") as configfile:
```

Comment:
Use the variable directly:

```suggestion
    with open(CONIFGPATH, "w") as configfile:
```